WordPressCom-Stats-iOS/StatsTableSectionHeaderView.m

```diff
@@ -18,18 +20,51 @@ - (instancetype)initWithReuseIdentifier:(NSString *)reuseIdentifier
         _theBoxView = [[UIView alloc] initWithFrame:CGRectZero];
         _theBoxView.backgroundColor = [UIColor colorWithRed:210.0/255.0 green:222.0/255.0 blue:238.0/255.0 alpha:1.0];
         [self.contentView addSubview:_theBoxView];
-        
     }
     return self;
 }
 
 
+- (void)setFrame:(CGRect)frame {
+    CGFloat width = self.superview.frame.size.width;
+    // On iPad, add a margin around tables
+    if (IS_IPAD && width > WPTableViewFixedWidth) {
+        CGFloat x = (width - WPTableViewFixedWidth) / 2;
```

Comment:
Maybe we should round up the value calculated in `x`. Floating positions might end up blurry'ing the view!